### PR TITLE
Mobile alignment issues-Career page

### PR DESCRIPTION
**Pull request by NikitK-deriv**
Changes:

-    fix few allignment issue for mobile response

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/chunks/section-label.tsx [hunk 1]
```diff
@@ -3,10 +3,14 @@ import styled from 'styled-components'
 import device from 'themes/device'
 import { CSSElementProps } from 'types/generics'
 
-const StyledSSHeader = styled.div`
+const StyledSSHeader = styled.div<{ jc?: boolean }>`
```
Comment (prince-deriv) [inline src/components/chunks/section-label.tsx:6]: Can we make this `string` instead.

File: src/components/chunks/section-label.tsx [hunk 2]
```diff
@@ -3,10 +3,14 @@ import styled from 'styled-components'
...
     display: flex;
     flex-direction: row;
     white-space: nowrap;
+
+    @media ${device.tabletL} {
+        justify-content: ${(props) => (props.jc ? 'center' : 'unset')};
```
Comment (prince-deriv) [inline src/components/chunks/section-label.tsx:12]: Make a value that we pass like `center` etc

File: src/components/chunks/section-label.tsx (hunk 1 above)
Comment (prince-deriv) [reply src/components/chunks/section-label.tsx:6]: You can also create a new entry on our `generics.ts` so we use it globally

File: src/components/chunks/section-label.tsx (hunk 2 above)
Comment (NikitK-deriv) [reply src/components/chunks/section-label.tsx]: No need in in realization with `props.jc || 'unset'`

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**